core/src/test/scala/com/avsystem/commons/concurrent/TaskExtensionsTest.scala

```diff
@@ -0,0 +1,52 @@
+package com.avsystem.commons
+package concurrent
+
+import monix.eval.Task
+import monix.execution.Scheduler
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.funsuite.AnyFunSuite
+import org.scalatest.matchers.should.Matchers
+import org.scalatestplus.scalacheck.ScalaCheckDrivenPropertyChecks
+
+import scala.concurrent.TimeoutException
+import scala.concurrent.duration._
+
+class TaskExtensionsTest extends AnyFunSuite with Matchers with ScalaCheckDrivenPropertyChecks with ScalaFutures {
+  import com.avsystem.commons.concurrent.TaskExtensions._
+
+  private implicit val scheduler: Scheduler = Scheduler.global
+
+  test("lazyTimeout") {
```

Comment:
This test does work in ScalaJs :( Any ideas how to fix it?

Reply:
This comment is still true and this should work as an async test spec: https://github.com/scalatest/scalatest/issues/2074#issuecomment-980492759

At least it does in Udash :)

Reply:
Moved to JVM